### DISCUSSION TITLE
How to get crop images location in original images

### DISCUSSION THREAD

**Discussion by XIANYUNYEHE-DEL** [Project-MONAI/MONAI#5524] [Q&A] [answered]
Now I have a crop image from `RandCropByPosNegLabeld`, and I want to get its location in original image. Where can I get this information?

**Comment by KumoLiu** [comment, marked as answer]
Hi @XIANYUNYEHE-DEL, you can get the `crop_center` like this:
```
ret = RandCropByPosNegLabeld(keys='image', label_key='label', spatial_size=(270,270))(data)
print(ret[0]['image'].meta['crop_center'])
```
Hope it can help you!
Thanks!

**Reply by XIANYUNYEHE-DEL** [reply]
Thank you for your answer, it solved my problem.
